SCHEDULE.md

```diff
@@ -0,0 +1,38 @@
+IPFS Workshop Schedule
+======================
+
+## Day 1
+
+> Get everyone psyched with IPFS, IPLD and libp2p. Get in depth understanding through hands on hacking.
+
+| Time   | Type     | Session                    | Speaker
+|:-------|:---------|:---------------------------|:-------
+| 10:00h | --       | people arrive and set up   | --
+| 10:15h | talk     | welcome presentation       | Juan or David
+| 10:30h | talk     | The State of IPFS          | Juan
+| 11:30h | talk     | IPLD                       | Juan
+| 12:15h | talk     | libp2p                     | David and Jeromy
+| 13:00h | open     | Lunch Break                | --
+| 14:15h | workshop | go-ipfs hands on           | Jeromy
+| 15:30h | workshop | IPLD hands on              | Juan and Friedel(remote)
+| 16:45h | workshop | libp2p hands on            | David and Jeromy
+| 18:00h | --       | end of day 1               | --
```

Comment:
I think the workshop should be about doing work, not presenting. I think there's way too many presentations. 

I hoped this event was more creating space for a lot of IPFS folks to get together and work on things together that require people's attention -- having important discussions for development. Not intro talks.

There's a ton of things that have been brought up in the last few days to be discussed during the workshop. Let's do that instead.